server/server_test.go

```diff
@@ -106,6 +106,9 @@ func (cli *testServerClient) getDSN(overriders ...configOverrider) string {
 // runTests runs tests using the default database `test`.
 func (cli *testServerClient) runTests(t *testing.T, overrider configOverrider, tests ...func(dbt *testkit.DBTestKit)) {
 	db, err := sql.Open("mysql", cli.getDSN(overrider))
+	db.SetMaxOpenConns(2)
+	db.SetMaxIdleConns(2)
+	db.SetConnMaxLifetime(10 * time.Second)
```

Comment:
These values are unlimited by default. Why do you set them?

Reply:
removed